microsoft/devhome
# Register AppInstaller at app launch

## Summary of the pull request
- Register `AppInstaller` when starting Dev Home (if not already registered)
- Moved initialization logic to `AppManagementInitializer`

## References and relevant issues

## Detailed description of the pull request / Additional comments

## Validation steps performed
- Tested with a staged package of AppInstaller:
  - AppInstaller was registered successfully when starting Dev Home and `App Install` was enabled
- Tested with AppInstaller package not on the system:
  - `App Install` was disabled (expected)
- Added Unit Test

## PR checklist
- [ ] Closes #465
